gcc/testsuite/rust/compile/negative_impls.rs

```diff
@@ -0,0 +1,10 @@
+// { dg-output "" }
```

Comment:
since this is a `compile` test, and not an `execute` one (it's in `gcc/testsuite/rust/compile/` rather than `gcc/testsuite/rust/execute`) there is no need to check the `stdout` output with `dg-output` - so you can remove that line :)

Reply:
rebase edited